packages/dart/test/relation_test.dart

```diff
@@ -0,0 +1,101 @@
+import 'dart:convert';
+
+import 'package:parse_server_sdk/parse_server_sdk.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('Relation', () {
+    test('addRelation', () async {
+      // arrange
+      await Parse().initialize(
```

Comment:
This section does not need to be written twice, write it once in the setup section, it is enough to take a look at [this section](https://github.com/parse-community/Parse-SDK-Flutter/blob/b07ce8c7fbe709333a6a97ce5473e6d5c88f33f0/packages/dart/test/parse_query_test.dart#L12)

Reply:
I'll do it right away, thank you very much for guiding me through the corrections, and thanks for your work